mmdet/ops/__init__.py

```diff
@@ -1,8 +1,10 @@
+from .affine_grid_generator import affine_grid
```

Comment:
`affine_grid` and `grid_sample` are currently unused. We may remove it from `ops/__init__.py` to speedup the loading of mmdet.